### ISSUE TITLE
Add support for Genes & Diseases on compare endpoints

### ISSUE THREAD

**Issue by kevinschaper** [monarch-initiative/monarch-app#290]
The V2 semantic similarity endpoint had support for expanding disease or gene IDs, so we should match that in V3. 

I'm guessing we might want to run an entity `get` on each ID provided, and then we can expand genes & phenotypes. 

We might also want to make a new response class that extends TermSetPairwiseSimilarity to make the phenotype expansions explicit. 

I'm also not sure how we should deal with negated phenotypes in the expansion.

**Comment by putmantime** [comment]
@kevinschaper Is this currently being done in the front end?

**Comment by kevinschaper** [comment]
@putmantime Yeah, this expansion happens on the front end. This is just support to have the API perform the same expansion.
